homeassistant/components/fibaro/climate.py

```diff
@@ -180,9 +182,9 @@ def __init__(self, fibaro_device):
             self._attr_temperature_unit = TEMP_CELSIUS
 
         if self._fan_mode_device:
-            fan_modes = (
-                self._fan_mode_device.fibaro_device.properties.supportedModes.split(",")
-            )
+            fan_modes = self._fan_mode_device.fibaro_device.properties.get(
+                "supportedModes"
+            ).split(",")
```

Comment:
I'd recommend using a library like pydantic in pyfibaro to parse the responses to typed python objects so the library user doesn't need to know the API protocol strings and do further parsing like splitting strings etc.